### PR TITLE
[WIP] New changes/hacks to work with libgdf_draft_csv_reader

### PR DESCRIPTION
**Pull request by Iroy30**

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by kkraus14** [comment]
add to whitelist

File: pygdf/io.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     for i in range(ncols):
+         newcol = Column.from_cffi_view(out[i])
+         # XXX: Bit mask is wrong? dropping it for now
+         newcol = newcol.replace(mask=None)
+         outcols.append(newcol.view(NumericalColumn, dtype=newcol.dtype))
+
+     # Build dataframe
+     df = DataFrame()
+     for k, v in zip(names, outcols):
+         df[k] = v
```
Comment (scopatz) [inline pygdf/io.py]: wouldn't it be more efficient to say `df = DataFrame(zip(names, outcols))`?

**Comment by scopatz** [comment]
Thanks for putting this in @Iroy30! Would you please add some tests too?

**Pull request closed without merging**